docs/pipeline-components-and-applications/loaders-storage-targets/bigquery-loader/index.md

```diff
@@ -68,10 +68,10 @@ In order to load this data again from `failedInserts` to BigQuery you can use
 
 Repeater has several important behavior aspects:
 
-- If a pulled record is not a valid Snowplow event, it will result into a `loader_recovery_error` bad row.
+- If a pulled record is not a valid Snowplow event, it will result into a `loader_recovery_error` failed event.
 - If a pulled record is a valid event, Repeater will wait some time (15 minutes by default) after the `etl_tstamp` before attempting to re-insert it, in order to let Mutator do its job.
-- If the database responds with an error, the row will get transformed into a `loader_recovery_error` bad row.
-- All entities in the dead-letter bucket are valid Snowplow [bad rows](https://github.com/snowplow/snowplow-badrows).
+- If the database responds with an error, the row will get transformed into a `loader_recovery_error` failed event.
```

Comment:
Note: docs for very old versions are deleted because I don’t have the patience to address this again and again :D